examples/basic-otlp/src/main.rs

```diff
@@ -23,18 +23,38 @@ fn init_tracer() -> Result<sdktrace::Tracer, TraceError> {
 
 // Skip first immediate tick from tokio, not needed for async_std.
 fn delayed_interval(duration: Duration) -> impl Stream<Item = tokio::time::Instant> {
-    opentelemetry::util::tokio_interval_stream(duration).skip(1)
+   opentelemetry::util::tokio_interval_stream(duration).skip(1)
+}
+
+use opentelemetry::metrics::Descriptor;
+use opentelemetry::sdk::export::metrics::Aggregator;
+use opentelemetry::sdk::export::metrics::AggregatorSelector;
+use std::sync::Arc;
```

Comment:
Maybe move this part to the top to keep the imports at the same place?

Reply:
moved to a different example and moved to the top